stylesheets/manifest.css

```diff
@@ -450,7 +451,7 @@ img.emoji {
     display: block; }
 
 .conversation-header {
-  border-bottom: solid 1px #d9d9d9;
+  border-bottom: solid 1px #dadada;
```

Comment:
This is weird. No change in the scss files that corresponds to this. I guess your darken() function darkens more than mine?